package.json

```
@@ -197,6 +202,11 @@
             }
         ],
         "keybindings": [
+            {
+                "command": "ahk++.alignSelection",
+                "key": "shift+win+f",
+                "when": "editorLangId == ahk"
```

Don't we also need the editor to have a selection here?

```suggestion
                "when": "editorLangId == ahk" && editorHasSelection
```